spec/plans/util/retrieve_and_upload_spec.rb

```diff
@@ -5,7 +5,7 @@
   include BoltSpec::Plans
 
   it 'file needs downloaded and needs uploaded' do
-    expect_task('peadm::os_identification')
+    expect_task('facts')
```

Comment:
I'm sure that we need to mock the facts task, but I'm not sure how at the moment :thinking: . And I'm not sure how the tests passed previously. Wasn't peadm::os_identification mocked any where?